examples/demo_app/app/javascript/packs/application.js

```diff
@@ -3,5 +3,5 @@ import Rails from '@rails/ujs';
 import Turbolinks from 'turbolinks';
 
 Rails.start();
-ActiveStorage.start();
 Turbolinks.start();
+ActiveStorage.start();
```

Comment:
This change doesn't really matter, but in new Rails 6 apps, `ActiveStorage`'s `start` method is called last.